db-migrations/5_log_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS log_game_round (
+	id serial PRIMARY KEY,
+    game_label VARCHAR(255) NOT NULL,
+    player_name VARCHAR(255) NOT NULL,
+    cards_played JSON NOT NULL,
+    cards_purchased JSON NOT NULL
```

Comment:
Instead of just the id being a primary key, we can do the following:
https://stackoverflow.com/questions/24050111/postgresql-composite-primary-keys-no-unique-constraint-matching-given-keys
`PRIMARY KEY (id, game_label, player_name)`